torch/serialization.py

```diff
@@ -397,10 +437,27 @@ def default_restore_location(storage, location):
 
 
 def normalize_storage_type(storage_type):
+    """Normalize a given storage type by retrieving the corresponding attribute from the `torch` module.
```

Comment:
I think it is okay to omit this docstring from this PR since it was not one of the pydocstyle errors in the issue